docs/development/remote-development.md

```diff
@@ -38,6 +38,11 @@ You can use [Gitpod](https://gitpod.io/) for light development work like:
 - Editing the docs
 - Running ESLint, Prettier
 
+To invoke renovte:
+
+- Define Gitpod [Environment Variable](https://gitpod.io/user/variables) RENOVATE_TOKEN. Populate it with your GitHub Personal Access Token (PAT)
+- Run: `yarn run start`
+
 For proper development, use GitHub Codespaces.
```

Comment:
```suggestion
```

We can drop this line once we confirm that Gitpod and GitHub Codespaces behave the same when developing Renovate.